### ISSUE TITLE
boundry issue when focus element

### ISSUE THREAD

**Issue by engrsandeep** [CurriculumAssociates/createjs-accessibility#43]
@mattbargar-cainc @mnewcomb314

When publish adobe animate file as image assets or spritesheet, then element get rendered on wrong position which shows element on wrong position when screen reader read the text.

Is there any solution for this?

**Comment by mnewcomb314** [comment]
There are a couple of possible causes:
1. If you are doing something like `const img = new createjs.Bitmap("test.png");` to add the image file to the canvas, before the image loads, it's bounds won't be able to be determined.  Whenever CAM can't determine the bounds, it gives it a width and height of 1px and a left/top of 0px relative to the a11y tree parent's coordinate space.  Once the file does load, then the bounds can be determined and the correct size and coordinates should be used, so it's important to have CAM do a render pass after stage redraws (for example https://github.com/CurriculumAssociates/createjs-accessibility-tester/blob/main/src/index.js#L17-L20).
2. Another possibility is if the canvas gets resized, in which case the DOM translation area needs to be resized to match.  That can be accomplished by calling `AccessibilityModule.resize` when resizing occurs.
3. While CAM presently handles scaling within the stage via CreateJS, if the canvas element's width and height attributes are set to different values than the canvas element's actual bounds (such as react-adobe-animate's example which has the attributes set to 500 but due to element's inline style and parent div sizing, it is actually 400px by 400px), then the canvas will be scaled in a way CAM doesn't currently account for.  That will lead to the bounds for the image to be close but not quite right.  I will look into getting this fixed, but it may take some time to find time to work on this.  A temporary work around for this is to ensure the width and height attributes match the dimensions that CSS specifies.

**Comment by mnewcomb314** [comment]
The wrong position of the element due to canvas scaling from differing HTML attribute width/height vs what CSS will lead to the width/height is fixed in https://github.com/CurriculumAssociates/createjs-accessibility/releases/tag/v1.0.3.

**Comment by mnewcomb314** [comment]
@drosenberg123 This was fixed in 1.0.3 and it looks like I just forgot to close this issue.
